geom/geom/src/TGeoParallelWorld.cxx

```diff
@@ -143,6 +163,14 @@ void TGeoParallelWorld::ResetOverlaps() const
 
 Bool_t TGeoParallelWorld::CloseGeometry()
 {
+   if (getenv("TGEO_PW_USEBVH")) {
```

Comment:
Since this is a runtime option, I would prefer to pass it as static property of the `TGeoManager` class (similar to for example setting `TGeoManager::fgVerboseLevel`